dcpy/lifecycle/scripts/validate_ingest.py

```diff
@@ -340,3 +373,28 @@ def _get_columns(
 
     for _, row in res.iterrows():
         print(row["t"])
+
+
+@app.command("load_from_s3")
+def _load_from_s3(
```

Comment:
I think data_loader will do all this already? Should pick up on your dev_bucket env var.